### PR TITLE
23 search ingredients

### LINKED ISSUE #18: Fix autocomplete bug
Steps to reproduce

- Open create/edit ingredients form
- Type anything in preparations
- Select any preparation. It doesn't get selected.

We need to fix this autocomplete or create a new one.

### LINKED ISSUE #23: Search Ingredients

### PR DESCRIPTION
**Pull request by Van-sh**
- Add Search in Ingredients
- Add API_URL constant in contants.ts as a configurable API base URL value
- Fix delete modal on mobile

### CODE REVIEW

**Comment by Van-sh** [comment]
This PR fulfills #23 and #21

**Comment by Van-sh** [comment]
Fixed #18

File: src/constants.ts [hunk 1]
```diff
@@ -1 +1,2 @@
-export const isDesktop = window.innerWidth > 640;
\ No newline at end of file
+export const isDesktop = window.innerWidth > 640;
+export const API_URL = "https://grocery-planner-be.onrender.com"
```
Comment (SakshiShreya) [inline src/constants.ts]: Add new line at EOF.

File: src/planner/ingredients/index.tsx [hunk 2]
```diff
@@ -71,13 +73,16 @@ export default function Ingredients() {
...
         <h1 className="text-2xl">Ingredients</h1>
+        <Search name="Ingredients" query={query} setQuery={setQuery} />
 
         {isLoading && <Loader />}
         {onGetSuccess &&
           (data.length === 0 ? (
             <BlankScreen name="Ingredients" onAdd={onEditModalOpen} />
           ) : (
-            <List data={data} onEdit={handleEdit} onDelete={showDeleteModal} />
+            <>
```
Comment (SakshiShreya) [inline src/planner/ingredients/index.tsx]: Remove fragment

**Review by SakshiShreya — CHANGES_REQUESTED** [review]

**Review by SakshiShreya — APPROVED** [review]

**Pull request merged**